SygicMapsKit/Classes/NavigationModule/SYMKNavigationView.swift

```diff
@@ -131,6 +137,22 @@ public class SYMKNavigationView: UIView {
         instructionTrailingConstraint = instructionView.trailingAnchor.constraint(equalTo: safeTrailingAnchor, constant: -margin)
         updateInstructionLayoutConstraints()
     }
+    
+    /// Setup speed cobntrol view for navigation module.
+    ///
+    /// - Parameter speedControlView: View with current speed and speed limit.
+    public func setupSpeedControlView(_ speedControlView: SYUISpeedControlView) {
+        self.speedControlView = speedControlView
+        addSubview(speedControlView)
+        speedControlView.translatesAutoresizingMaskIntoConstraints = false
+        if let infobarView = infobarView {
+            speedControlTrailingPortraitContraint = speedControlView.bottomAnchor.constraint(equalTo: infobarView.topAnchor, constant: -margin)
+        } else {
+            speedControlTrailingPortraitContraint = speedControlView.bottomAnchor.constraint(equalTo: safeBottomAnchor, constant: -margin)
+        }
+        speedControlTrailingLandscapeContraint = speedControlView.bottomAnchor.constraint(equalTo: safeBottomAnchor, constant: -margin)
+        speedControlView.trailingAnchor.constraint(equalTo: safeTrailingAnchor, constant: -margin).isActive = true
+    }
```

Comment:
na konci volaj updateSpeedControlLayoutConstraints()

Reply:
jj, zabudol som to zavolat